server/eb/README.md

```diff
@@ -116,9 +113,9 @@ about_legal_privacy: /static/cellxgene/deploy/privacy.html
 Additional scripts can be added using the server/inline_scripts config parameters.
 To include these scripts in the deployment, use the following steps:
 
-* In this directory, create a sub directory called "customize/inline_scripts".
-* Copy the script files into this directory
-* modify your configuration file to set the location to these file (leaving off customize/inline_scripts)
+- In this directory, create a sub directory called "customize/inline_scripts".
+- Copy the script files into this directory
+- Modify your configuration file to set the location to these file (leaving off customize/inline_scripts)
```

Comment:
Do you think it would be worth adding a file structure diagram containing all of the directories/files customization options?

Reply:
Good idea, I just added a section